.github/workflows/release.yml

```diff
@@ -57,20 +73,28 @@ jobs:
 
             - uses: actions/setup-node@v4
               with:
-                  node-version: ${{ env.NODE_VERSION }}
+                  node-version: ${{ needs.get-node-version.outputs.node-version }}
                   registry-url: https://registry.npmjs.org
               env:
                   NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
             - name: Install curl
               run: sudo apt-get install curl -y
 
-            - name: Install global dependencies
-              run: ./.github/scripts/install_global_dependencies.sh
+            - id: get-dfx-version
```

Comment:
I am leaning towards this version but as long as they are here I might as well make sure that seems right

Reply:
I like it